Packages/Sources/RootView/Home.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+import NodeCryptoCore
+import SwiftUI
+
+struct RootView: View {
+  @State private var activeTab: Tab = .home
+  @Namespace var animation
+
+  var body: some View {
+    VStack(spacing: 0) {
+      TabView(selection: $activeTab) {
+        Color.red
+          .tag(Tab.home)
+          .toolbar(.hidden, for: .tabBar)
+
+        Text("Services")
+          .tag(Tab.search)
+          .toolbar(.hidden, for: .tabBar)
+
+        Text("Notifications")
+          .tag(Tab.notifications)
+          .toolbar(.hidden, for: .tabBar)
+
+        Text("Profile")
+          .tag(Tab.profile)
+          .toolbar(.hidden, for: .tabBar)
+      }
+
+      customTabBar()
+    }
+  }
+
+  @ViewBuilder
+  func customTabBar() -> some View {
+    HStack(spacing: 0) {
+
```

Comment:
You are putting so many empty lines everywhere. It is not necessary for readability. Please, remove all empty vertical lines within methods or closures. I promise you will quickly get used to this dense code. Empty lines enhance readability only in long methods, which should be avoided anyway. Whenever you think you should add empty line, it can be a hint that the code should be extracted to a new method. The more context you can see at once, the better for you and readers of your code. Some people work on 14" macbooks, and want to see code, not spaces. Even on biggest screens the more code you can see, the more context you have.

Reply:
I see,I didn't think of it that way.
I'll remove the spaces.